### DISCUSSION TITLE
Single request to multiple destinations

### DISCUSSION THREAD

**Discussion by czobrisky** [dotnet/yarp#1516] [Q&A] [answered]
Currently using YARP and had a requirement to broadcast a request to multiple destinations. Is this possible with YARP?

**Comment by adityamandaleeka** [comment, marked as answer]
Is this a duplicate of https://github.com/microsoft/reverse-proxy/issues/105 ?

**Comment by czobrisky** [comment]
It does look like a duplicate. Will comment on that thread.
